src/shared/lib/getDefaultMolecularProfiles.ts

```diff
@@ -123,32 +117,23 @@ export function getDefaultGeneSetProfile(profiles: MolecularProfile[]) {
 
 export function getFilteredMolecularProfiles(
     profiles: MolecularProfile[],
-    profileFilterSet?: { [profileType: string]: boolean },
+    queriedProfileSuffixes?: string[],
     dataPriority: number = 0 // this is legacy and necessary only for backward compatibility
```

Comment:
@jjgao do you think we can drop support for dataPriority at this point?

Reply:
@alisman I think we cannot. there might be old/saved urls which still uses it